static/css/devhub/base.less

```diff
@@ -105,6 +108,29 @@ section {
   }
 }
 
+.DevHub-Navigation--signedin {
+  background-color: @secondary-section-background;
+
+  ul {
+    li {
+      margin: 10px 10px;
+
+      a {
+        color: @secondary-font-color;
+      }
+
+      .Logo {
```

Comment:
This looks a bit over nested - this is going to result in some pretty long selectors e.g:

`.DevHub-Navigation--signedin ul li .Logo`

See if you can extract the Logo bit out to be on its own. Generally if nesting is more than 2 levels deep it's worth looking at again.